telos-frontend/src/components/top-bar/TopBar.jsx

```diff
@@ -0,0 +1,73 @@
+import AppBar from '@material-ui/core/AppBar';
+import { makeStyles } from '@material-ui/core/styles';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import SettingsIcon from '@material-ui/icons/Settings';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+
+// Used to define styles of MUI components.
+const useStyles = makeStyles(() => ({
+  root: {
+    backgroundColor: '#7A00F4',
+  },
+  rightSide: {
+    marginLeft: 'auto',
+    marginRight: -12,
+  },
+  userButton: {
+    marginLeft: 15,
+  },
+}));
+
```

Comment:
A brief comment here explaining what the component is would be good.

Reply:
Ok